test/cri-containerd/policy_test.go

```diff
@@ -64,18 +64,20 @@ func policyFromOpts(t *testing.T, policyType string, opts ...securitypolicy.Poli
 		config.AllowRuntimeLogging,
 		config.AllowEnvironmentVariableDropping,
 		config.AllowUnencryptedScratch,
+		config.AllowCapabilityDropping,
 	)
 	if err != nil {
 		t.Fatal(err)
 	}
 	return base64.StdEncoding.EncodeToString([]byte(policyString))
 }
 
-func alpineSecurityPolicy(t *testing.T, policyType string, allowEnvironmentVariableDropping bool, opts ...securitypolicy.ContainerConfigOpt) string {
+func alpineSecurityPolicy(t *testing.T, policyType string, allowEnvironmentVariableDropping bool, allowCapabilityDropping bool, opts ...securitypolicy.ContainerConfigOpt) string {
```

Comment:
can we introduce a `securitypolicy.ContainerConfigOpt` that will set the `AllowPrivilegeEscalation` value on the container config?

Reply:
i dont understand the question at all. Sorry.

Reply:
Talked offline. Nothing to change here.